pkg/enemy/hitlag.go

```diff
@@ -66,6 +66,6 @@ func (e *Enemy) Tick() {
 	}
 	e.timePassed += left
 
-	e.Reactable.Tick()
 	queue.Run(&e.queue, e.timePassed)
+	e.Reactable.Tick()
```

Comment:
this is to ensure that 8 ticks at 20 durability burning fuel are possible without messing with durability/decay rate